2024-07-25 12:55:14,739 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Current SDK version is 0.17.5
2024-07-25 12:55:14,740 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Configure stats pid to 246692
2024-07-25 12:55:14,740 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-07-25 12:55:14,740 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Loading settings from /content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/settings
2024-07-25 12:55:14,740 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'varnet-sweep-test', 'entity': 'superfastmri-seoul-national-university', 'root_dir': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge', 'sweep_id': 'a690ua88', 'run_id': '3569vk5v', 'sweep_param_path': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/sweep-a690ua88/config-3569vk5v.yaml'}
2024-07-25 12:55:14,740 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-25 12:55:14,741 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/train.py', 'program': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/train.py'}
2024-07-25 12:55:14,741 INFO    Thread-3 (_run_job):246692 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-25 12:55:14,741 INFO    Thread-3 (_run_job):246692 [wandb_init.py:_log_setup():529] Logging user logs to /content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/run-20240725_125514-3569vk5v/logs/debug.log
2024-07-25 12:55:14,741 INFO    Thread-3 (_run_job):246692 [wandb_init.py:_log_setup():530] Logging internal logs to /content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/run-20240725_125514-3569vk5v/logs/debug-internal.log
2024-07-25 12:55:14,741 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():569] calling init triggers
2024-07-25 12:55:14,741 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():576] wandb.init called with sweep_config: {'cascade': 12, 'chans': 9, 'sens_chans': 5}
config: {}
2024-07-25 12:55:14,742 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():619] starting backend
2024-07-25 12:55:14,742 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():623] setting up manager
2024-07-25 12:55:14,751 INFO    Thread-3 (_run_job):246692 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-07-25 12:55:14,752 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():631] backend started and connected
2024-07-25 12:55:14,770 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_config_callback():1382] config_cb None None {'cascade': 12, 'chans': 9, 'sens_chans': 5}
2024-07-25 12:55:14,771 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():720] updated telemetry
2024-07-25 12:55:17,635 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-07-25 12:55:18,011 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_on_init():2435] communicating current version
2024-07-25 12:55:18,108 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_on_init():2444] got version response 
2024-07-25 12:55:18,108 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():804] starting run threads in backend
2024-07-25 12:55:18,580 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_console_start():2413] atexit reg
2024-07-25 12:55:18,580 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_redirect():2255] redirect: wrap_raw
2024-07-25 12:55:18,584 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_redirect():2320] Wrapping output streams.
2024-07-25 12:55:18,584 INFO    Thread-3 (_run_job):246692 [wandb_run.py:_redirect():2345] Redirects installed.
2024-07-25 12:55:18,588 INFO    Thread-3 (_run_job):246692 [wandb_init.py:init():847] run started, returning control to user process
2024-07-25 12:55:33,128 WARNING MsgRouterThr:246692 [router.py:message_loop():77] message_loop has been closed
2024-07-25 12:55:34,898 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Current SDK version is 0.17.5
2024-07-25 12:55:34,900 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Configure stats pid to 246692
2024-07-25 12:55:34,900 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-07-25 12:55:34,900 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Loading settings from /content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/settings
2024-07-25 12:55:34,900 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'varnet-sweep-test', 'entity': 'superfastmri-seoul-national-university', 'root_dir': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge', 'run_id': 'p7qvj8cz', 'sweep_param_path': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/sweep-a690ua88/config-p7qvj8cz.yaml', 'sweep_id': 'a690ua88'}
2024-07-25 12:55:34,900 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-25 12:55:34,900 INFO    Thread-6 (_run_job):246692 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/train.py', 'program': '/content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/train.py'}
2024-07-25 12:55:34,901 INFO    Thread-6 (_run_job):246692 [wandb_init.py:_log_setup():529] Logging user logs to /content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/run-20240725_125534-p7qvj8cz/logs/debug.log
2024-07-25 12:55:34,901 INFO    Thread-6 (_run_job):246692 [wandb_init.py:_log_setup():530] Logging internal logs to /content/drive/MyDrive/fastMRI_main_TMAttFIVarNet/FastMRI_challenge/wandb/run-20240725_125534-p7qvj8cz/logs/debug-internal.log
2024-07-25 12:55:34,901 INFO    Thread-6 (_run_job):246692 [wandb_init.py:init():569] calling init triggers
2024-07-25 12:55:34,901 INFO    Thread-6 (_run_job):246692 [wandb_init.py:init():576] wandb.init called with sweep_config: {'cascade': 12, 'chans': 11, 'sens_chans': 5}
config: {}
2024-07-25 12:55:34,901 INFO    Thread-6 (_run_job):246692 [wandb_init.py:init():614] wandb.init() called when a run is still active
2024-07-25 12:55:34,902 ERROR   Thread-6 (_run_job):246692 [wandb_init.py:init():1191] error in wandb.init()
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/wandb_init.py", line 1181, in init
    return wi.init()
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/wandb_init.py", line 615, in init
    with telemetry.context() as tel:
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/lib/telemetry.py", line 42, in __exit__
    self._run._telemetry_callback(self._obj)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/wandb_run.py", line 798, in _telemetry_callback
    self._telemetry_flush()
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/wandb_run.py", line 809, in _telemetry_flush
    self._backend.interface._publish_telemetry(self._telemetry_obj)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/interface/interface_shared.py", line 101, in _publish_telemetry
    self._publish(rec)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2024-07-25 12:55:34,980 INFO    Thread-6 (_run_job):246692 [wandb_run.py:_finish():2107] finishing run superfastmri-seoul-national-university/varnet-sweep-test/3569vk5v
2024-07-25 12:55:40,521 INFO    Thread-8 (_run_job):246692 [wandb_run.py:_finish():2107] finishing run superfastmri-seoul-national-university/varnet-sweep-test/3569vk5v
2024-07-25 12:55:45,956 INFO    Thread-9 (_run_job):246692 [wandb_run.py:_finish():2107] finishing run superfastmri-seoul-national-university/varnet-sweep-test/3569vk5v
2024-07-25 12:55:51,348 INFO    Thread-10 (_run_job):246692 [wandb_run.py:_finish():2107] finishing run superfastmri-seoul-national-university/varnet-sweep-test/3569vk5v
